scene/resources/skeleton_modification_2d_twoboneik.cpp

```diff
@@ -172,13 +177,22 @@ void SkeletonModification2DTwoBoneIK::_execute(float p_delta) {
 
 		if (isnan(angle_0) || isnan(angle_1)) {
 			// We cannot solve for this angle! Do nothing to avoid setting the rotation (and scale) to NaN.
-		} else {
+		} else if (same_scale_sign) {
 			joint_one_bone->set_global_rotation(angle_atan - angle_0 - joint_one_bone->get_bone_angle());
 			joint_two_bone->set_rotation(-Math_PI - angle_1 - joint_two_bone->get_bone_angle() + joint_one_bone->get_bone_angle());
+		} else {
+			joint_one_bone->set_global_rotation(angle_atan + angle_0 + joint_one_bone->get_bone_angle());
+			joint_two_bone->set_rotation(-Math_PI - angle_1 - joint_two_bone->get_bone_angle() + joint_one_bone->get_bone_angle());
```

Comment:
OK, but it feels like you could keep the general else block from before commit then handle both cases at once:
- for the first line, a -1/+1 multiplier would do the trick
- the second line is always the same, could be preserved at end of else block

Reply:
About the first point, you mean creating a variable to hold -1 ou 1, them multiplying `angle_0 + joint_one_bone->get_bone_angle()` by the variable? Can you show me what you mean?  

About the second point, like this?
```cpp
if (isnan(angle_0) || isnan(angle_1)) {
	// We cannot solve for this angle! Do nothing to avoid setting the rotation (and scale) to NaN.
} else {
	if (same_scale_sign) {
		joint_one_bone->set_global_rotation(angle_atan - angle_0 - joint_one_bone->get_bone_angle());
	} else {
		joint_one_bone->set_global_rotation(angle_atan + angle_0 + joint_one_bone->get_bone_angle());
	}
	
	joint_two_bone->set_rotation(-Math_PI - angle_1 - joint_two_bone->get_bone_angle() + joint_one_bone->get_bone_angle());
}
```

Reply:
Yes, using a float multiplier like this:
```cpp
float sign_multiplier = same_scale_sign ? -1.0f : 1.0f;
joint_one_bone->set_global_rotation(angle_atan + sign_multiplier * (angle_0 + joint_one_bone->get_bone_angle()));
```

Reply:
But looking at your comment below, `else` looks fine too.